merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/CreateSimulationDatasetAction.java

```diff
@@ -44,33 +44,34 @@ public SimulationDatasetRecord apply(
     PreparedStatements.setTimestamp(this.statement, 2, simulationStart);
     PreparedStatements.setTimestamp(this.statement, 3, planStart);
 
-    final var results = this.statement.executeQuery();
-    if (!results.next()) throw new FailedInsertException("simulation_dataset");
-    final Status status;
-    try {
-      status = Status.fromString(results.getString(2));
-    } catch (final Status.InvalidSimulationStatusException ex) {
-      throw new Error("Simulation Dataset initialized with invalid state.");
-    }
+    try(final var results = this.statement.executeQuery()) {
```

Comment:
This change is fine, but note from the `ResultSet` javadocs:

> A ResultSet object is automatically closed when the Statement object that generated it is closed, re-executed, or used to retrieve the next result from a sequence of multiple results.

So this is at worst a *bounded* leak, since the next call to `apply` will automatically release the previously-generated `ResultSet`.

Reply:
Astute. Happy to undo if we think there's value in say readability or something like that.

Reply:
No, I think this is fine. More noting that, if you're looking for leaks that eventually lead to destabilization of the system, this probably isn't one. But it's good to address anyway.

Reply:
👍